docs/getting-started/integrations-cdn/next-js.md

```diff
@@ -17,8 +17,110 @@ CKEditor&nbsp;5 does not support server-side rendering yet, but you can integrat
 
 {@snippet getting-started/use-builder}
 
-You get ready-to-use code tailored to your needs! You can take the output from the builder, specifically the npm React snippet, and follow the npm path below. Just replace the content of the `components/custom-editor.js` file. The snippet may contain client-side hooks, so do not forget about adding the `'use client'` directive in the case of the App Router.
-
 ## Setting up the project
 
 This guide assumes you already have a Next project. To create such a project, you can use CLI like `create-next-app`. Refer to the [Next.js documentation](https://nextjs.org/docs/app/api-reference/create-next-app) to learn more.
+
+## Using from CDN
+
+First, install the CKEditor 5 packages:
+
+* `ckeditor5` &ndash; package with open-source plugins and features.
+* `ckeditor5-premium-features` &ndash; package with premium plugins and features.
+
+Depending on your configuration and selected plugins, you may need to install only the first one or both.
+
+```bash
+npm install ckeditor5 ckeditor5-premium-features
+```
+
+Next.js is based on React, so install the [CKEditor 5 WYSIWYG editor component for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react), too:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
```

Comment:
I'd note that these dependencies are only needed for typings and if Next.js in JavaScript mode is used then there is no need to install these packages.